react/tic-tac-toe/src/app/screens/Game/components/TableMatches/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { arrayOf } from 'prop-types';
+
+import actionsCreators from '../../../../../redux/table-matches/actions';
+import Match from '../Match';
+import styles from './styles.module.scss';
+import Spinner from 'react-spinkit';
+
+class TableMatches extends Component {
+  componentDidMount() {
+    this.props.getMatches();
+  }
+
+  renderMatch = (info) => (<Match match={info} key={info.id} />);
+
+  render () {
+    const { matches } = this.props;
+    return matches.length ? 
+    (<div className={styles.matches}>
+        <div>Historial de Partidas:</div>
+        <table className={styles.tableMatches}>
+          <thead>
+            <tr>
+              <th className={styles.tableChild}>Player One</th>
+              <th className={styles.tableChild}>Player Two</th>
+              <th className={styles.tableChild}>Winner</th> 
+            </tr>
+          </thead>
+          <tbody>{this.props.matches.map(this.renderMatch)}</tbody>
```

Comment:
You already destructured props so
```jsx
<tbody>{matches.map(this.renderMatch)}</tbody>
```
also, this could actually be a separate component

Reply:
The array of Matches a separate component?